anime_downloader/sites/helpers/selescrape.py

```diff
@@ -0,0 +1,156 @@
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+from anime_downloader.const import get_random_header
+from selenium.webdriver.common.by import By
+from urllib.parse import urlencode
+from urllib.parse import urlsplit
+from selenium import webdriver
+from bs4 import BeautifulSoup
+from logging import exception
+from sys import platform
+import requests, os
+import click
+import time
+import json
+
+header = get_random_header()
+def test_os():
+    if platform.startswith("linux"):
+        os_name = "linux"
+    elif platform == "darwin":
+        os_name = "darwin"
+    elif platform == "win32":
+        os_name = "win32"
+    return os_name
+gg = test_os()
+if gg.startswith("linux"):
+    dd = '\\'
+    aa = '/'
+elif gg() == 'win32':
+    dd = '/'
+    aa = '\\'
+else:
+    raise exception('This OS is not currently supported.')
```

Comment:
this will make the tool unusable with mac

Reply:
Well, I set as a priority to support the popular systems first. For Mac users I'll work on it today and tomorrow to add support for them.

Reply:
the tool on mac is usable, but only the supported browsers in the script. As of now it is Chrome and Firefox.

Reply:
that's fine